ads/amoad.js

```diff
@@ -21,13 +21,26 @@ import {loadScript, checkData, validateDataExists} from '../3p/3p';
  * @param {!Object} data
  */
 export function amoad(global, data) {
-  checkData(data, ['sid']);
+  checkData(data, ['sid', 'adType']);
   validateDataExists(data, ['sid']);
 
+  let script;
+  const attrs = {};
+  if (data['adType'] === 'native') {
+    script = 'https://j.amoad.com/js/n.js';
+    attrs['class'] = 'amoad_native';
+    attrs['data-sid'] = data.sid;
+  } else {
+    script = 'https://j.amoad.com/js/a.js';
+    attrs['class'] = `amoad_frame sid_${data.sid} container_div sp`;
+  }
+  global.amoadOption = {ampData: data};
```

Comment:
by default we won't mangle public properties (anything that doesn't end with `_`) so it is now safe to no longer declare this in ads.extern.js, just a nice to have. (this used to not be the case)

LGTM